scripts/vote_dummy.py

```diff
@@ -0,0 +1,64 @@
+"""
+Just test voting
+"""
+
+import time
+
+from typing import (Dict, Tuple, Optional, List)
+
+from brownie.network.transaction import TransactionReceipt
+
+from utils.voting import confirm_vote_script, create_vote
+from utils.evm_script import encode_call_script
+
+from utils.config import (get_deployer_account, get_is_live, contracts)
+from utils.permissions import encode_permission_create, encode_permission_grant_p, encode_permission_revoke
+from utils.permission_parameters import Param, Op, ArgumentValue
+
+# noinspection PyUnresolvedReferences
+from utils.brownie_prelude import *
+
+
+def encode_set_elrewards_withdrawal_limit(limit_bp: int) -> Tuple[str, str]:
+    lido: interface.Lido = contracts.lido
+    return lido.address, lido.setELRewardsWithdrawalLimit.encode_input(limit_bp)
+
+
+def start_vote(
+    tx_params: Dict[str, str],
+    silent: bool = False,
+) -> Tuple[int, Optional[TransactionReceipt]]:
+    """Prepare and run voting."""
+    voting: interface.Voting = contracts.voting
+    lido: interface.Lido = contracts.lido
+    self_owned_steth_burner: interface.SelfOwnedStETHBurner = contracts.self_owned_steth_burner
+
+    encoded_call_script = encode_call_script([
+        # 4. Set Execution Layer rewards withdrawal limit to 2BP
+        encode_set_elrewards_withdrawal_limit(2),
```

Comment:
Oops, it's strange to have such a thing there.
As far as I know, voting allows passing even completely empty scripts